packages/lit-dev-content/site/docs/v3/releases/upgrade.md

```diff
@@ -29,6 +29,66 @@ For the vast majority of users there should be no required code changes to upgra
 
 Lit 2.x and 3.0 are _interoperable_ – templates, base classes, and directives from one version of Lit will work with those from another.
 
+## Lit is now published as ES2021
+
+Lit 2 was published as ES2019, and Lit 3 is now published as ES2021. ES2021 is supported by modern browsers.
+This may be a breaking change for your tooling if it can't parse ES2021, for example in the case of Webpack 4.
+
+### Using Lit 3 with Webpack 4
+
+Webpack 4's internal parser doesn't support nullish coalescing (`??=`), or optional chaining (`?.`), which are features of ES2021.
+When Webpack 4 parses Lit 3, or other modern packages, it'll throw a `Module parse failed: Unexpected token` on parsing `??=` or `?.`.
```

Comment:
```suggestion
Webpack 4's internal parser doesn't support nullish coalescing (`??`), logical assignment (`??=`), or optional chaining (`?.`), which are syntaxes introduced in ES2021 so will throw a `Module parse failed: Unexpected token` when encountering those.
```

Reply:
Adding this great change. I think some of these were also technically introduced in ES2020, [`?.` for example](https://262.ecma-international.org/11.0/#prod-OptionalChain). ES2020 also breaks Webpack 4.